xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 8.4.0
+JSONRPC_VERSION 8.4.1
```

Comment:
I think this should be 8.5.0, as it is an API addition

Reply:
you bump 
- major on complete redesign
- minor on API breakage
- patch on everything else

so then you apply other rules than the rest of the world

Reply:
Sorry, I don't think that sounds right

Reply:
i cba to fight over this, you will have your 8.5.0. but you *ARE* wrong.

Reply:
For future see https://github.com/notspiff/xbmc/blob/1fdfd4d4b5c9a02a4fec94b3eb6a9576a421d316/xbmc/interfaces/json-rpc/schema/methods.json#L33

Reply:
you do apply other rules than the rest of the world.

Reply:
Entirely possible, they are not my rules. Sorry I didn't spot immediately and we had this tennis

Reply:
nw. i should have spotted it as well, assumptions are the mother of all evil. and i should stop using 'you' and start using 'we'. habits take some time to change..